src/playground/src/components/playground/ImageCard.tsx

```diff
@@ -30,133 +35,243 @@ const useStyles = makeStyles({
+          if (newValue) {
+            setGenerating(false);
+          } else {
+            setGenerating(true);
+          }
+          setPrompt("");
+        }}
+      >
+        <option value="">Select a model</option>
+        {eventData &&
+          eventData.capabilities["openai-dalle3"] &&
+          eventData.capabilities["openai-dalle3"].map((model) => (
+            <option key={model} value={model}>
+              {model}
+            </option>
+          ))}
+      </Select>
+
       <Label
         style={{ fontSize: "medium", marginBottom: "0.5rem" }}
         htmlFor={promptId}
       >
         <strong>Prompt</strong>
+        <Tooltip content="Describe the image you want to create. For example, 'watercolor painting of the Seattle skyline'" relationship="description" >
+          <Info16Filled className={styles.tooltip} />
+        </Tooltip>
       </Label>
-      <Input
-        type="text"
-        size="medium"
+
+      <Textarea
+        style={{ width: "90%" }}
         id={promptId}
         value={prompt}
-        onChange={(e) => setPrompt(e.target.value)}
-        placeholder="Enter a prompt. Eg: cute picture of an cat"
+        disabled={isGenerating}
+        placeholder="Enter a prompt. Eg: cute picture of an cat (Shift + Enter for new line)"
+        onChange={(e) => {
+          setPrompt(e.currentTarget.value);
+        }}
+        onKeyDown={(e) => {
+          if (e.key === "Enter" && !e.shiftKey && prompt) {
+            generateImage(prompt);
+            setGenerating(true);
+            e.preventDefault();
+          }
+        }}
       />
-      <Button
-        onClick={() => generateImage(prompt)}
-        disabled={!prompt || !canGenerate}
-      >
-        Generate
-      </Button>
-    </div>
+      <div>
+        <Button
+          onClick={() => {
+            if (!isGenerating) {
+              generateImage(prompt)
+              setGenerating(true);
+            }
+          }}
+          disabled={!prompt || isGenerating}
+          className={styles.smallButton}
+          icon={<SendRegular />}
+          appearance="primary"
+          style={{ textAlign: "left", marginBottom: "12px", marginTop: "12px" }}
+        >
+          Generate
+        </Button>
+        <Button
+          onClick={() => {
+            if (!isGenerating) {
+              setPrompt("");
+            }
+          }}
+          disabled={!prompt || isGenerating}
+          className={styles.smallButton}
+          id="clear-button"
+          icon={<Delete24Regular />}
+          iconPosition="before"
+        >
+          Clear prompt
+        </Button>
+      </div>
+    </div >
   );
 };
 
-const ImageList = ({ images }: { images: ExtendedImageGenerations[] }) => {
-  const styles = useStyles();
+const ImageList = ({ images, isGenerating, setGenerating }: { images: ExtendedImageGenerations[], isGenerating: boolean, setGenerating: Dispatch<React.SetStateAction<boolean>> }) => {
```

Comment:
Extract the props type as a TypeScript type rather than inlining

Reply:
not sure what this means?